cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/ExpressionHandler.kt

```diff
@@ -85,6 +85,15 @@ class ExpressionHandler(frontend: PythonLanguageFrontend) :
         }
     }
 
+    private fun handleNamedExpr(node: Python.AST.NamedExpr): Expression {
+        return newAssignExpression(
```

Comment:
You will probably need to set `usedAsExpression` to true. This is a switch whether the `AssignExpression` is used as a statement (false) or an as expression (true)